### PR TITLE
[PIS-33] Add endpoints to accept and reject order by restaurant

### PR DESCRIPTION
**Pull request by wkol**
https://linear.app/pis22z/issue/PIS-33/be

### CODE REVIEW

File: application/src/main/kotlin/dev/hypest/pis/infrastructure/db/DbOrderQuery.kt [hunk 1]
```diff
@@ -43,11 +47,25 @@ class DbOrderQuery(
...
     }
+
+    private fun getStatus(
+        draftOrder: DraftOrderEntity,
+        orderToPay: OrderToPayEntity?,
+        activeOrder: ActiveOrderEntity?,
+        orderDelivery: OrderDeliveryEntity?
+    ) = when {
+//        orderDelivery?.isDelivered -> OrderResponse.OrderStatus.DELIVERED
+//        orderDelivery?.isBeingDelivered -> OrderResponse.OrderStatus.DELIVERY_IN_PROGRESS
```
Comment (BartlomiejRasztabiga) [inline application/src/main/kotlin/dev/hypest/pis/infrastructure/db/DbOrderQuery.kt:61]: to chyba mozna odkomentowac (w sensie isBeingDelivered)

**Review by BartlomiejRasztabiga — APPROVED** [review]

**Comment by BartlomiejRasztabiga** [comment]
/home/runner/work/backend/backend/application/src/main/kotlin/dev/hypest/pis/payments/adapter/in/event/OrderRejectedEventListener.kt:1:1: The package declaration does not match the actual file location. [InvalidPackageDeclaration]

File: application/src/main/kotlin/dev/hypest/pis/infrastructure/db/DbOrderQuery.kt (hunk 1 above)
Comment (wkol) [reply application/src/main/kotlin/dev/hypest/pis/infrastructure/db/DbOrderQuery.kt:61]: Tylko Twoje najpierw zmergować i sobie maina zrebasuje

Comment (BartlomiejRasztabiga) [reply application/src/main/kotlin/dev/hypest/pis/infrastructure/db/DbOrderQuery.kt:61]: a faktoza xD już merguje
EDIT: done

**Pull request merged**